sources/osgGA/OrbitManipulator.js

```diff
@@ -261,6 +261,11 @@ utils.createPrototypeObject(
             };
         })(),
 
+        _computePitch: function(prevPitch, dy) {
```

Comment:
name could be misleading ?
_computeYaw and _computePitch not in the same level (_computeYaw not called in code above,  not same functionnalities), but exact same name convention ?